src/BlazorUI/Bit.BlazorUI/Components/Progress/Loading/BitSpinnerLoading.scss

```
@@ -19,10 +16,12 @@
     position: absolute;
     border-radius: 20%;
     top: var(--bit-ldn-spn-3);
-    left: var(--bit-ldn-spn-37);
-    width: var(--bit-ldn-spn-6);
     height: var(--bit-ldn-spn-18);
     background: var(--bit-ldn-color);
+    width: var(--bit-ldn-stroke, var(--bit-ldn-spn-6));
+    // Centred against the box rather than offset by a measured amount, so that a bar made thicker or
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)